Refactor FMFeature

s9latimm opened the pull request:
Replace `vara::Feature` with `vara::feature::Feature`.

s9latimm committed:
Refactor

s9latimm committed:
Update feature class

s9latimm committed:
Add TODO

s9latimm committed:
Remove include

s9latimm committed:
Update pybind11 and fix tidy

s9latimm committed:
Add python hash

s9latimm committed:
Remove values

s9latimm committed:
Merge branch 'vara-dev' into f-RefactorFMFeature

vulder commented on include/vara/Feature/Feature.h:
```diff
@@ -20,20 +25,27 @@ namespace vara::feature {
 //===----------------------------------------------------------------------===//
 
-/// \brief Base class for components of \a FeatureModel.
+/// \brief Base class for components of \a FeatureModel or \a FeatureRegion.
```
```suggestion
/// \brief Base class for components of \a FeatureModel.
```
We don't need to mention this here. The usage of VaRA should be "decoupled" from the feature lib.

vulder reviewed (approved):
lgtm

s9latimm committed:
Update include/vara/Feature/Feature.h

Co-authored-by: Florian Sattler <vuld3r@gmail.com>

s9latimm committed:
Rename hash

M	bindings/python/vara-feature/pybind_Feature.cpp
M	include/vara/Feature/Feature.h

s9latimm committed:
Reformat

M	bindings/python/vara-feature/pybind_Feature.cpp

s9latimm committed:
Merge branch 'vara-dev' into f-RefactorFMFeature

M	README.md
M	bindings/python/vara-feature/CMakeLists.txt
M	tools/fm-viewer/CMakeLists.txt
M	unittests/CMakeLists.txt

s9latimm committed:
Merge branch 'f-RefactorFMFeature' of github.com:se-passau/vara-feature into f-RefactorFMFeature

M	include/vara/Feature/Feature.h
